src/helper/remote_config/protocol/tuf/client_get_configs_request.hpp

```diff
@@ -0,0 +1,33 @@
+// Unless explicitly stated otherwise all files in this repository are
+// dual-licensed under the Apache-2.0 License or BSD-3-Clause License.
+//
+// This product includes software developed at Datadog
+// (https://www.datadoghq.com/). Copyright 2021 Datadog, Inc.
+#pragma once
+
+#include <list>
+
+#include "../cached_target_files.hpp"
+#include "../client.hpp"
+
+namespace dds::remote_config::protocol::tuf {
+
+//@todo: Remove client from the name
+class client_get_configs_request {
+public:
+    client_get_configs_request(
+        client client, std::list<cached_target_files> cached_target_files)
```

Comment:
Name clash here, consider adding the namespace to the type or use a different name for the parameter.

Reply:
Done